packages/spec-test-runner/test/spec/altair/fork_choice/get_head_minimal.test.ts

```diff
@@ -0,0 +1,207 @@
+  isAttestation,
+  isBlock,
+  isCheck,
+} from "./type";
+import {LodestarForkChoice} from "@chainsafe/lodestar/lib/chain/forkChoice/forkChoice";
+import {SPEC_TEST_LOCATION} from "../../../utils/specTestCases";
+import {ChainEventEmitter} from "@chainsafe/lodestar/lib/chain/emitter";
+import {toHexString} from "@chainsafe/ssz";
+import {createIBeaconConfig} from "@chainsafe/lodestar-config";
+import {CheckpointStateCache} from "@chainsafe/lodestar/lib/chain/stateCache/stateContextCheckpointsCache";
+import {IForkChoice} from "@chainsafe/lodestar-fork-choice";
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+const config = createIBeaconConfig({...params, ALTAIR_FORK_EPOCH: 0});
+
+describeDirectorySpecTest<IForkChoiceTestCase, void>(
+  "forkchoice get_head",
+  join(SPEC_TEST_LOCATION, "/tests/minimal/altair/fork_choice/get_head/pyspec_tests"),
+  (testcase) => {
+    const emitter = new ChainEventEmitter();
+    const {steps, anchorState} = testcase;
+    const currentSlot = anchorState.slot;
+    const tbState = config.getForkTypes(currentSlot).BeaconState.createTreeBackedFromStruct(anchorState);
+    let wrappedState = createCachedBeaconState(config, tbState);
+    const forkchoice = new LodestarForkChoice({config, emitter, currentSlot, state: wrappedState});
+    const checkpointStateCache = new CheckpointStateCache(config);
+    const stateCache = new Map<string, CachedBeaconState<allForks.BeaconState>>();
+    cacheState(wrappedState, stateCache);
+    const {SECONDS_PER_SLOT} = wrappedState.config.params;
+    for (const step of steps) {
+      if (isTick(step)) {
+        forkchoice.updateTime(Number(step.tick) / SECONDS_PER_SLOT);
+      } else if (isAttestation(step)) {
+        const attestation = testcase.attestations.get(step.attestation);
+        if (!attestation) throw Error(`No attestation ${step.attestation}`);
+        forkchoice.onAttestation(wrappedState.epochCtx.getIndexedAttestation(attestation));
+      } else if (isBlock(step)) {
+        const signedBlock = testcase.blocks.get(step.block);
+        if (!signedBlock) throw Error(`No block ${step.block}`);
+        expect(signedBlock).not.to.be.undefined;
+        const preState = stateCache.get(toHexString(signedBlock.message.parentRoot));
+        if (!preState)
+          throw new Error("not found parent state for parent root" + toHexString(signedBlock.message.parentRoot));
+        wrappedState = runStateTranstion(preState, signedBlock, forkchoice, checkpointStateCache);
+        cacheState(wrappedState, stateCache);
+      } else if (isCheck(step)) {
+        const {
+          head: expectedHead,
+          time: expectedTime,
+          justifiedCheckpointRoot,
+          finalizedCheckpointRoot,
+          bestJustifiedCheckpoint,
+        } = step.checks;
+        const head = forkchoice.getHead();
+        expect(head.slot).to.be.equal(Number(expectedHead.slot));
+        expect(toHexString(head.blockRoot)).to.be.equal(expectedHead.root);
```

Comment:
Please add very detailed assertion messages to all the expect() calls in this function, i.e.
```
Wrong expected head root at step ${step.id}
```
Otherwise debugging this is very hard